src/config.rs

```diff
@@ -67,7 +67,7 @@ pub const PLATFORM_BOOT_HART_ID: usize =
     parse_usize_or(option_env!("MIRALIS_PLATFORM_BOOT_HART_ID"), 0);
 
 /// Whether any benchmark is enabled
```

Comment:
Small nit: the doc comment should be updated.

Reply:
:)